src/obr/core/queries.py

```diff
@@ -14,6 +19,18 @@ class query_result:
     result: list[dict] = field(default_factory=list[dict])
     sub_keys: list[list[str]] = field(default_factory=list[list[str]])
 
+    def __repr__(self) -> str:
+        return f"Job: {self.id}, result: {self.result}"
+
+
+class Predicates(Enum):
+    eq = "=="
+    neq = "!="
+    geq = ">="
+    gt = ">"
+    leq = "<="
+    lt = "<"
+
 
 @dataclass
 class Query:
```

Comment:
At some point I think we should build some kind of type hierarchy here. Like having a `Query` and a `FilterQuery` class

Reply:
We can definitely discuss this in the separate PR regarding the query (syntax) overhaul!